storybook/CMakeLists.txt

```diff
@@ -17,18 +17,11 @@ find_package(
 
 file(GLOB_RECURSE QML_FILES "stubs/*.qml" "Storybook/*.qml" "../ui/StatusQ/*.qml" "../ui/app/*.qml")
 file(GLOB_RECURSE JS_FILES "../ui/StatusQ/*.js" "../ui/app/*.js")
-file(GLOB_RECURSE HEADERS *.h)
-if(APPLE)
-  file(GLOB_RECURSE SOURCES *.cpp *.mm)
-else()
-  file(GLOB_RECURSE SOURCES *.cpp)
-endif()
 
 add_executable(
```

Comment:
@osmaczko can you add also this ? W/o this we can't build for Mac:
```
if (APPLE)
    set(MACOS_VERSION_MIN_FLAGS -mmacosx-version-min=10.14)
    set(CMAKE_OSX_ARCHITECTURES "x86_64")
endif()
```
^ @noeliaSD

Reply:
> @osmaczko can you add also this ? W/o this we can't build for Mac:
> 
> ```
> if (APPLE)
>     set(MACOS_VERSION_MIN_FLAGS -mmacosx-version-min=10.14)
>     set(CMAKE_OSX_ARCHITECTURES "x86_64")
> endif()
> ```
> 
> ^ @noeliaSD

Tested on mac with this addition!

Reply:
Guys, this is git, you can add/push the change to the branch yourselves ;)

Reply:
Added, but this change request was strange :sweat_smile: I can't test it, because I am not on mac.

Reply:
Exactly what I'd suggested above :D

Reply:
You should trust us.. ;)